Plugins/Public/event/Main.cpp

```diff
@@ -64,7 +66,10 @@ struct TRADE_EVENT {
 	uint uHashID;
 	string sEventName;
 	string sURL;
+	string eventDescription;
```

Comment:
Please turn into wstring, and call stows only on config load instead of every time it's read in line 709.
If possible, extend that to the other strings in the struct as well.